tests/testsuite/artifact_dep.rs

```diff
@@ -2085,17 +2192,16 @@ fn env_vars_and_build_products_for_various_build_targets() {
         .build();
     p.cargo("test -Z bindeps")
         .masquerade_as_nightly_cargo(&["bindeps"])
-        .with_stderr(
-            "\
+        .with_stderr_data(str![[r#"
 [LOCKING] 2 packages to latest compatible versions
-[COMPILING] bar [..]
-[COMPILING] foo [..]
-[FINISHED] `test` profile [unoptimized + debuginfo] target(s) in [..]
-[RUNNING] unittests [..]
-[RUNNING] tests/main.rs [..]
+[COMPILING] bar v0.5.0 ([ROOT]/foo/bar)
+[COMPILING] foo v0.0.0 ([ROOT]/foo)
+[FINISHED] `test` profile [unoptimized + debuginfo] target(s) in [ELAPSED]s
+[RUNNING] unittests src/lib.rs (target/debug/deps/foo-[..])
+[RUNNING] tests/main.rs (target/debug/deps/main-[..])
```

Comment:
Should we try to have an auto-redaction for these?

Reply:
Taken care of in #14054